boa/src/syntax/ast/op.rs

```diff
@@ -66,8 +67,14 @@ pub enum UnaryOp {
     Not,
     /// `~a` - bitwise-not of the value
     Tilde,
+    /// `type of` - Get the type of object
```

Comment:
Isn't the operator `typeof`? 
```suggestion
    /// `typeof` - Get the type of object
```